server/app/views/applicant/TrustedIntermediaryClientListView.java

```diff
@@ -241,7 +241,7 @@ private DivTag renderClientList(
   }
 
   private LiTag renderClientCard(AccountModel account, Messages messages) {
-    return li().withClass("usa-card tablet-lg:grid-col-6 widescreen:grid-col-4")
```

Comment:
comment (non-blocking):  These tablet and widescreen styles came from the USWDS card example here [card](https://designsystem.digital.gov/components/card/).  Their cards are narrower, so that made sense there, but you're right that, at least here on the TI dashboard, the intention is for the cards to take up the whole width.